### ISSUE TITLE
[train-133] 11 errors on server tests {latest,stage}

### ISSUE THREAD

**Issue by jrgm** [mozilla/fxa-content-server#7066]
https://tc-test.dev.lcip.org/viewLog.html?buildId=48918&tab=buildResultsDiv&buildTypeId=fxa_LatestServerTests

#get /.well-known/fxa-client-configuration 
- returns a JSON doc with expected values

node: routes/get-metrics-flow remote request (10)

- invalid context query parameter |  
- invalid entrypoint query parameter |  
- invalid form_type query parameter |  
- invalid service query parameter |  
- invalid utm_campaign query parameter |  
- invalid utm_content query parameter |  
- invalid utm_medium query parameter |  
- invalid utm_source query parameter |  
- invalid utm_term query parameter |  
- valid query parameters

**Comment by shane-tomlinson** [comment]
> #get /.well-known/fxa-client-configuration
> returns a JSON doc with expected values

It looks like the channel server is not what's expected:
```
A wss://dev.channelserver.nonprod.cloudops.mozgcp.net
E wss://channelserver.services.mozilla.com
```

> node: routes/get-metrics-flow remote request (10)

These are returning a 401 unauthorized, which indicates either the Origin header isn't being set, or if it is, the value being sent is not part of the [allow list](https://github.com/mozilla/fxa-content-server/blob/master/server/lib/configuration.js#L21) for that route.

**Comment by shane-tomlinson** [comment]
> These are returning a 401 unauthorized, which indicates either the Origin header isn't being set, or if it is, the value being sent is not part of the allow list for that route.

On latest, the list is:
> ALLOWED_METRICS_FLOW_ORIGINS=null,http://127.0.0.1:8001,http://localhost:8000,http://127.0.0.1:8000,https://www.mozilla.org,https://www.allizom.org,https://www-demo5.allizom.org,https://www-demo4.allizom.org,https://www-demo3.allizom.org,https://www-dev.allizom.org

The test runner tries to use the first origin in [allowed_metrics_flow_cors_origin](https://github.com/mozilla/fxa-content-server/blob/10f1ba62099335ddbc48e0e6929a0c1563b72eef/server/lib/configuration.js#L21), which won't have any defined on teamcity. I added ALLOWED_METRICS_FLOW_ORIGINS=null to the environment variables for the teamcity project, let's see if that fixes the problem!

**Comment by shane-tomlinson** [comment]
>  I added ALLOWED_METRICS_FLOW_ORIGINS=null to the environment variables for the teamcity project, let's see if that fixes the problem!

Yup - https://tc-test.dev.lcip.org/viewLog.html?buildId=48927&tab=buildResultsDiv&buildTypeId=fxa_LatestServerTests, only the fxa-client-configuration error remains.

**Comment by shane-tomlinson** [comment]
Fixed the last error too by adding PAIRING_SERVER_BASE_URI set to the development value. Here's where @vladikoff can say "that's what I had checked in to the default config originally, bonehead."
